uniflow/flow/flow_factory.py

```diff
@@ -10,18 +10,18 @@ class FlowFactory:
     _flows = {EXTRACT: {}, TRANSFORM: {}, RATER: {}}
 
     @classmethod
-    def register(cls, name: str, flow_cls: "Flow") -> None:  # noqa: F821
+    def register(cls, name: str, flow_cls: """Flow""") -> None:  # noqa: F821
```

Comment:
We usually use single quote for parameter names. Can you reverse back for the whole file?